src/main/resources/db/migration/V1_18__staff_location_correction.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE correction_request add column location varchar(20);
```

Comment:
This will need to be made not nullable once migrated again

Reply:
while we still need to figure out the business needs here *first*, i think it ought to remain nullable